security-admin/src/main/webapp/scripts/routers/Router.js

```diff
@@ -45,23 +46,23 @@ function(Backbone, Marionette, localization, MAppState, XAUtil){
 			"!/userprofile"		: "userProfileAction",
 			
 			"!/users/:tab"		: "userManagerAction",
-			"!/user/create"		: "userCreateAction",
+			"!/user/create"		: RangerConfigMgr.isUserGroupManagementEnabled()? "userCreateAction":"pageNotFoundAction",
 			"!/user/:id"		: "userEditAction",
 			
-			"!/group/create"	: "groupCreateAction",
+			"!/group/create"	: RangerConfigMgr.isUserGroupManagementEnabled()? "groupCreateAction":"pageNotFoundAction",
 			"!/group/:id"		: "groupEditAction",
 
 			/************GENERIC UI *****************************************/
-			"!/service/:serviceType/create" 	: "serviceCreateAction",
-			"!/service/:serviceType/edit/:id"	: "serviceEditAction",
+			"!/service/:serviceType/create" 	: RangerConfigMgr.isServiceManagementEnabled()? "serviceCreateAction":"pageNotFoundAction",
+			"!/service/:serviceType/edit/:id"	: RangerConfigMgr.isServiceManagementEnabled()? "serviceEditAction":"pageNotFoundAction",
 			
 			"!/service/:serviceId/policies/:policyType"			: "policyManageAction",
 			"!/service/:serviceId/policies/create/:policyType"	: "RangerPolicyCreateAction",
 			"!/service/:serviceId/policies/:id/edit"			: "RangerPolicyEditAction",
 
 			/************PERMISSIONS VIEWS *****************************************/
-            "!/permissions"					: "modulePermissionsAction",
-            "!/permissions/:id/edit"        : "modulePermissionEditAction",
+			"!/permissions"					: "modulePermissionsAction",
```

Comment:
indentation?

Reply:
fixed and aligned with the next line in the section.